doc/manpages/dinit-service.5.m4

```diff
@@ -173,13 +173,19 @@ Variable substitution is not performed on the \fBenv\-file\fR property value its
 If the path is not absolute, it is resolved relative to the directory containing the service
 description.
 .TP
-\fBrestart\fR = {yes | true | no | false}
+\fBrestart\fR = {yes | true | on-failure | no | false}
 Indicates whether the service should automatically restart if it stops, including due to
-unexpected process termination or a dependency stopping.
+unexpected process termination or a dependency stopping; \fBon-failure\fR indicates auto-restart
+should only happen when exit code of process is not equal to 0 or signalled with any other
+signal than SIGHUP, SIGINT, SIGUSR1, SIGUSR2 or SIGTERM. the \fBon-failure\fR only works for
```

Comment:
Always use new line before beginning a new sentence in the man pages (see the README).

Grammar: letters at the start of a sentence should be capital (it should be "The" not "the").